cmd/clusterctl/examples/vsphere/machineset.yaml.template

```diff
@@ -22,9 +22,6 @@ spec:
           apiVersion: vsphere.cluster.k8s.io/v1alpha1
           kind: VsphereMachineProviderSpec
           datacenter: "${VSPHERE_DATACENTER}"
```

Comment:
@akutz  it make sense to me now, thx for the explanation, could you help to clarify why `datacenter` kept in both machine spec and cloud provider configure?

Reply:
Hi @figo,

Because you can configure multiple datacenter via multiple vCenter definitions in the cloud provider configuration. So technically you could have a K8s cluster with each of its control plane nodes in different datacenters as long as each datacenter had symmetrical pool/datastore/folder structures.